tile-floor.js

```diff
@@ -0,0 +1,45 @@
+const AFRAME = window.AFRAME;
+AFRAME.registerComponent('tile-floor', {
+  schema: {
+    tileColumnCount: {type: 'int', default: 15},
+    tileRowCount: {type: 'int', default: 15},
+    tileRadius: {type: 'int', default: 1.5}
+  },
+  init: function () {
+    console.log("In tile-floor component")
+  },
+  update: function () {
```

Comment:
Let's do this in init instead of update so that we know it only happens once. I guess if we did it in update it would change when we change the values right? maybe its fine this way...

In either case, use "let" instead of "var" (it's a new way to declare variables in JS since 2015 which causes less nonsense.